lib/liquid_voting/voting.ex

```diff
@@ -220,8 +220,48 @@ defmodule LiquidVoting.Voting do
       ** (Ecto.NoResultsError)
 
   """
-  def get_participant!(id, organization_uuid),
-    do: Repo.get_by!(Participant, id: id, organization_uuid: organization_uuid)
+  def get_participant!(uuid, organization_uuid),
+    do: Repo.get_by!(Participant, uuid: uuid, organization_uuid: organization_uuid)
+
+  @doc """
+  Gets a single participant for an organization uuid by their uuid
+
+  Returns nil if the Participant does not exist.
+
+  ## Examples
+
+      iex> get_participant_by_uuid("2b041b4e-81bd-4c6f-96db-0bac9b03b991", "a6158b19-6bf6-4457-9d13-ef8b141611b4")
+      %Participant{}
+
+      iex> get_participant_by_uuid("unregistered-45c1-4a65-af87-6cbbbafadb14", "a6158b19-6bf6-4457-9d13-ef8b141611b4")
+      nil
+
+  """
+
+  # TODO? Remove this function, as get_participant!/2, above, has same functionality?
```

Comment:
I'd say yes, but check dependencies on it.

beware the bangs, too - [Repo.get_by!](https://hexdocs.pm/ecto/Ecto.Repo.html#c:get_by!/3) raises an exception when it doesn't find a record, whereas [Repo.get_by](https://hexdocs.pm/ecto/Ecto.Repo.html#c:get_by/3) returns `nil`. So they're not exactly the same. The question of when to use exceptions might popup, and the answer is: it's complicated. In principle they should be only used when something unexpected happens, but there's grey areas and different schools of thought... on going discussion.

Reply:
so, we need get_participant with bang, without, or both?

Reply:
For this PR I'd just stick with what's being used currently. So if both versions are being used somewhere, keep both. If not, get rid of the zombie code

Reply:
I didn't do this... maybe move to separate issue?

Reply:
yes, a separate PR at least. not worth opening an issue for it